### PR TITLE
Added grep feature and updated readme

### PR DESCRIPTION
**Pull request by marchampson**
Added the grep feature and updated the README with how to use.

### CODE REVIEW

File: src/TailCommand.php [hunk 1]
```diff
@@ -79,6 +80,8 @@ protected function getEnvironmentConfiguration(string $environment): array
 
     public function getTailCommand(): string
     {
-        return 'tail -f -n '.$this->option('lines').' "`ls -t | head -1`"';
+        $grep = ($this->option('grep')) ? ' | grep "' . $this->option('grep').'"' : '';
```
Comment (freekmurze) [inline src/TailCommand.php]: Could you add a test for this?

Comment (marchampson) [reply src/TailCommand.php]: Sure, will take a look in the morning.

Comment (freekmurze) [reply src/TailCommand.php]: I got a little time, I'll do it now!

**Comment by freekmurze** [comment]
Thanks for your work on this!

**Pull request merged**